OCCE 6.9.0 uses not declared c_tries

I am trying to install tigl 3.0.0-rc2 with occ version 6.9.1, however it breaks with `tigl-3.0.0-rc2/src/common/tiglcommonfunctions.cpp:1331:21: error: 'c_tries' was not declared in this scope`
Looking into tiglcommonfunctions.cpp, c_tris only gets declared when the OCC Version >= 7.2.0:

```
#if OCC_VERSION_HEX >= VERSION_HEX_CODE(7,2,0)
    const int c_tries = 3;
#endif
```

but c_tries does still get used later on when the version is lower than that:

```
#if OCC_VERSION_HEX >= VERSION_HEX_CODE(7,2,0)
        if (splitter.HasErrors()) {
            if (i < c_tries - 1) {
                fuzzyValue *= 10;
                LOG(WARNING) << "SplitShape failed, retrying with fuzzyValue: " << fuzzyValue;
                continue;
            }

            std::ostringstream oss;
            splitter.GetReport()->Dump(oss);
            LOG(ERROR) << "unable to split passed shapes: " << oss.str();
            throw tigl::CTiglError("unable to split passed shapes: " + oss.str());
        }
#elif OCC_VERSION_HEX >= VERSION_HEX_CODE(6,9,0)
        if (splitter.ErrorStatus() != 0) {
            if (i < c_tries - 1) {
                fuzzyValue *= 10;
                LOG(WARNING) << "SplitShape failed, retrying with fuzzyValue: " << fuzzyValue;
                continue;
            }
```

--- a/src/common/tiglcommonfunctions.cpp
+++ b/src/common/tiglcommonfunctions.cpp
@@ -1290,9 +1290,6 @@ TopoDS_Shape SplitShape(const TopoDS_Shape& src, const TopoDS_Shape& tool)
 {
 #if OCC_VERSION_HEX >= VERSION_HEX_CODE(6,9,0)
     double fuzzyValue = Precision::Confusion();
-#endif
-
-#if OCC_VERSION_HEX >= VERSION_HEX_CODE(7,2,0)
     const int c_tries = 3;
 #endif
 

